2025-03-01 21:16:13,079 [INFO] 🚀 データ取得を開始
2025-03-01 21:16:13,079 [DEBUG] スクリプト開始
2025-03-01 21:16:13,079 [DEBUG] API_KEY: WGV3FY8PM9SKEZTK
2025-03-01 21:16:13,788 [DEBUG] API Response Status: 200
2025-03-01 21:16:13,789 [DEBUG] API Response Content: {
    "Meta Data": {
        "1. Information": "Daily Prices (open, high, low, close) and Volumes",
        "2. Symbol": "NVDA",
        "3. Last Refreshed": "2025-02-28",
        "4. Output Size": "Compact",
        "5. Time Zone": "US/Eastern"
    },
    "Time Series (Daily)": {
        "2025-02-28": {
            "1. open": "118.0200",
            "2. high": "125.0900",
            "3. low": "116.4000",
            "4. close": "124.9200",
            "5. volume": "389091145"
        },
      
2025-03-01 21:16:13,789 [DEBUG] API Request URL: https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=NVDA&apikey=WGV3FY8PM9SKEZTK&outputsize=compact&datatype=json
2025-03-01 21:16:13,807 [INFO] ✅ データ品質チェック OK！
2025-03-01 21:16:13,807 [DEBUG] データベース接続を開始
2025-03-01 21:16:13,836 [ERROR] ❌ データ保存エラー: could not translate host name "postgres_container" to address: nodename nor servname provided, or not known
Traceback (most recent call last):
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/tests/test_src/test_fetch.py", line 117, in save_to_db
    conn = psycopg2.connect(**DB_CONFIG)
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/venv/lib/python3.13/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres_container" to address: nodename nor servname provided, or not known

2025-03-01 21:16:13,837 [INFO] 🎉 処理完了
2025-03-01 21:45:29,487 [INFO] 🚀 データ取得を開始
2025-03-01 21:45:29,488 [DEBUG] スクリプト開始
2025-03-01 21:45:29,488 [DEBUG] API_KEY: WGV3FY8PM9SKEZTK
2025-03-01 21:45:30,221 [DEBUG] API Response Status: 200
2025-03-01 21:45:30,221 [DEBUG] API Response Content: {
    "Meta Data": {
        "1. Information": "Daily Prices (open, high, low, close) and Volumes",
        "2. Symbol": "NVDA",
        "3. Last Refreshed": "2025-02-28",
        "4. Output Size": "Compact",
        "5. Time Zone": "US/Eastern"
    },
    "Time Series (Daily)": {
        "2025-02-28": {
            "1. open": "118.0200",
            "2. high": "125.0900",
            "3. low": "116.4000",
            "4. close": "124.9200",
            "5. volume": "389091145"
        },
      
2025-03-01 21:45:30,221 [DEBUG] API Request URL: https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=NVDA&apikey=WGV3FY8PM9SKEZTK&outputsize=compact&datatype=json
2025-03-01 21:45:30,235 [INFO] ✅ データ品質チェック OK！
2025-03-01 21:45:30,235 [DEBUG] データベース接続を開始
2025-03-01 21:45:30,251 [ERROR] ❌ データ保存エラー: could not translate host name "postgres_container" to address: nodename nor servname provided, or not known
Traceback (most recent call last):
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/tests/test_src/test_fetch.py", line 117, in save_to_db
    conn = psycopg2.connect(**DB_CONFIG)
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/venv/lib/python3.13/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres_container" to address: nodename nor servname provided, or not known

2025-03-01 21:45:30,256 [INFO] 🎉 処理完了
2025-03-01 21:45:44,628 [INFO] 🚀 データ取得を開始
2025-03-01 21:45:44,628 [DEBUG] スクリプト開始
2025-03-01 21:45:44,628 [DEBUG] API_KEY: WGV3FY8PM9SKEZTK
2025-03-01 21:45:45,009 [DEBUG] API Response Status: 200
2025-03-01 21:45:45,009 [DEBUG] API Response Content: {
    "Meta Data": {
        "1. Information": "Daily Prices (open, high, low, close) and Volumes",
        "2. Symbol": "NVDA",
        "3. Last Refreshed": "2025-02-28",
        "4. Output Size": "Compact",
        "5. Time Zone": "US/Eastern"
    },
    "Time Series (Daily)": {
        "2025-02-28": {
            "1. open": "118.0200",
            "2. high": "125.0900",
            "3. low": "116.4000",
            "4. close": "124.9200",
            "5. volume": "389091145"
        },
      
2025-03-01 21:45:45,009 [DEBUG] API Request URL: https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=NVDA&apikey=WGV3FY8PM9SKEZTK&outputsize=compact&datatype=json
2025-03-01 21:45:45,023 [INFO] ✅ データ品質チェック OK！
2025-03-01 21:45:45,023 [DEBUG] データベース接続を開始
2025-03-01 21:45:45,143 [ERROR] ❌ データ保存エラー: connection to server at "localhost" (::1), port 5432 failed: FATAL:  role "myuser" does not exist
Traceback (most recent call last):
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/tests/test_src/test_fetch.py", line 117, in save_to_db
    conn = psycopg2.connect(**DB_CONFIG)
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/venv/lib/python3.13/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (::1), port 5432 failed: FATAL:  role "myuser" does not exist

2025-03-01 21:45:45,144 [INFO] 🎉 処理完了
2025-03-01 21:46:28,999 [INFO] 🚀 データ取得を開始
2025-03-01 21:46:28,999 [DEBUG] スクリプト開始
2025-03-01 21:46:28,999 [DEBUG] API_KEY: WGV3FY8PM9SKEZTK
2025-03-01 21:46:29,375 [DEBUG] API Response Status: 200
2025-03-01 21:46:29,375 [DEBUG] API Response Content: {
    "Meta Data": {
        "1. Information": "Daily Prices (open, high, low, close) and Volumes",
        "2. Symbol": "NVDA",
        "3. Last Refreshed": "2025-02-28",
        "4. Output Size": "Compact",
        "5. Time Zone": "US/Eastern"
    },
    "Time Series (Daily)": {
        "2025-02-28": {
            "1. open": "118.0200",
            "2. high": "125.0900",
            "3. low": "116.4000",
            "4. close": "124.9200",
            "5. volume": "389091145"
        },
      
2025-03-01 21:46:29,376 [DEBUG] API Request URL: https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=NVDA&apikey=WGV3FY8PM9SKEZTK&outputsize=compact&datatype=json
2025-03-01 21:46:29,397 [INFO] ✅ データ品質チェック OK！
2025-03-01 21:46:29,398 [DEBUG] データベース接続を開始
2025-03-01 21:46:29,419 [ERROR] ❌ データ保存エラー: connection to server at "localhost" (::1), port 5432 failed: FATAL:  role "myuser" does not exist
Traceback (most recent call last):
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/tests/test_src/test_fetch.py", line 117, in save_to_db
    conn = psycopg2.connect(**DB_CONFIG)
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/venv/lib/python3.13/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (::1), port 5432 failed: FATAL:  role "myuser" does not exist

2025-03-01 21:46:29,420 [INFO] 🎉 処理完了
2025-03-01 21:59:32,504 [INFO] 🚀 データ取得を開始
2025-03-01 21:59:32,504 [DEBUG] スクリプト開始
2025-03-01 21:59:32,504 [DEBUG] API_KEY: WGV3FY8PM9SKEZTK
2025-03-01 21:59:33,302 [DEBUG] API Response Status: 200
2025-03-01 21:59:33,302 [DEBUG] API Response Content: {
    "Meta Data": {
        "1. Information": "Daily Prices (open, high, low, close) and Volumes",
        "2. Symbol": "NVDA",
        "3. Last Refreshed": "2025-02-28",
        "4. Output Size": "Compact",
        "5. Time Zone": "US/Eastern"
    },
    "Time Series (Daily)": {
        "2025-02-28": {
            "1. open": "118.0200",
            "2. high": "125.0900",
            "3. low": "116.4000",
            "4. close": "124.9200",
            "5. volume": "389091145"
        },
      
2025-03-01 21:59:33,302 [DEBUG] API Request URL: https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=NVDA&apikey=WGV3FY8PM9SKEZTK&outputsize=compact&datatype=json
2025-03-01 21:59:33,319 [INFO] ✅ データ品質チェック OK！
2025-03-01 21:59:33,319 [DEBUG] データベース接続を開始
2025-03-01 21:59:33,514 [ERROR] ❌ データ保存エラー: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "my_database" does not exist
Traceback (most recent call last):
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/tests/test_src/test_fetch.py", line 117, in save_to_db
    conn = psycopg2.connect(**DB_CONFIG)
  File "/Users/koichi/Library/CloudStorage/GoogleDrive-yoshidome.kouichi@gmail.com/.shortcut-targets-by-id/1YPflACfNCBy1sNnuIrSAGpX5J8mL3Pcn/Google Drive/01.Private/Documents/Alpha_Vantage_project/venv/lib/python3.13/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "my_database" does not exist

2025-03-01 21:59:33,517 [INFO] 🎉 処理完了
